x-pack/test/security_solution_cypress/cypress/e2e/explore/hosts/vulnerabilities_contextual_flyout.cy.ts

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
Thanks for adding the test. Looks like it's interacting with alerts page and the flyout, shall we move it to `x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/expandable_flyout`?

Reply:
Agree